lua/cronex/cron_from_line.lua

```diff
@@ -67,4 +67,49 @@ M.cron_from_line = function(line)
     return nil
 end
 
+-- Patterns for standard crontab format (no quotes)
+local crontab_patterns = {
+    -- Standard 5-part cron expression (minute hour day month weekday)
+    "^%s*([%d%*%-%/,]+)%s+([%d%*%-%/,]+)%s+([%d%*%-%/,]+)%s+([%d%*%-%/,]+)%s+([%d%*%-%/,]+)%s",
+    -- Special time strings like @daily, @hourly, etc.
+    "^%s*(@%w+)%s",
```

Comment:
Sadly construe doesn't support this format. Do you know of a good cron explainer that does @fabridamicelli ? There are a few options in the Readme but none of them seem like a perfect fit.